src/seedu/addressbook/data/AddressBook.java

```diff
@@ -44,6 +48,8 @@ public AddressBook(UniquePersonList persons, UniqueTagList tags) {
         for (Person p : allPersons) {
             syncTagsWithMasterList(p);
         }
+        
+        allTaggings= new ArrayList<Tagging>();
```

Comment:
why is allTaggings getting initialized twice? One tagging list is enough for a session.